Networking/Networking/Model/ShippingLabel/Packages/ShippingLabelCustomsForm.swift

```diff
@@ -4,6 +4,13 @@ import Codegen
 /// Represents customs info for a shipping label package
 ///
 public struct ShippingLabelCustomsForm: Equatable, GeneratedFakeable {
+    /// ID of the associated package.
+    ///
+    /// This is for identifying the package when inputing customs form only,
+    /// no need for encoding and sending to remote.
+    ///
+    public let packageID: String
```

Comment:
It makes sense to me. Not sure what will represent a `packageID` at the moment, because it should be if I'm right, an internal implementation.

Reply:
That's correct - it's an internal implementation. I'm not quite happy with this, but I have yet to come up with a better solution.